src/forces/Gravity.re

```diff
@@ -0,0 +1,16 @@
+let g = 6.67428 *. Math.pow(~base=10, ~exp=-11);
+let force = (~m1, ~m2, ~r) => g *. m1 *. m2 /. Math.sqf(r);
+
+let forceV = (~m1, ~m2, ~v1, ~v2) => {
+  // Derive the vector pointing from m1 to m2.
+  let v = Vector.subf(~v1, ~v2);
```

Comment:
Super nit, is `Vector.subf(~v1, ~v2)` equivalent to `v1 - v2`?  If so, the resultant vector will be a vector that points from v2 -> v1, but your comment says it's a vector pointing from m1 -> m2.  

This may be irrelevant, but if the formula requires a vector that points from m1 -> m2, then the vector subtraction should be `Vector.subf(~v2, ~v1)`.

And if the it doesn't matter for the formula, then at least the comment should say it's a vectoring "pointing from m2 to m1" :D 

Here's a pic below to illustrate what I mean:
![image](https://user-images.githubusercontent.com/40646372/70265007-6190f800-174e-11ea-8008-d32ba9c7f8af.png)

Reply:
Sharp as always @mariano-formidable! Thanks for pointing that out. You're right that the comment should get switched around. It doesn't matter too much (yet) for our purposes that `subf` is implemented as `v1 - v2`, but it is good to be precise in the comments. Fixing now!